store/window.js

```diff
@@ -1,4 +1,8 @@
 module.exports = function (state, emitter) {
+  state.window = {
```

Comment:
All store functions should set its state on the corresponding scope, not directly to `state`.